gammapy/maps/tests/test_wcsnd.py

```diff
@@ -586,3 +592,44 @@ def test_get_spectrum():
 
     spec = m.get_spectrum(region=region, func=np.mean)
     assert_allclose(spec.data, [1.0, 1.0, 1.0])
+
+
+def get_npred_map():
+    position = SkyCoord(0.0, 0.0, frame="galactic", unit="deg")
+    energy_axis = MapAxis.from_bounds(
+        1, 100, nbin=30, unit="TeV", name="energy", interp="log"
+    )
+
+    exposure = Map.create(
+        binsz=0.02,
+        map_type="wcs",
+        skydir=position,
+        width="5 deg",
```

Comment:
Maybe decrease a bit the `width` here to e.g. `2 deg`, so that the test runs faster...